addon/model.js

```diff
@@ -80,6 +80,10 @@ let megamorphicModelProxyHandler, megamorphicNativeDeprecationHandler;
 
 if (CUSTOM_MODEL_CLASS) {
   const MegamorphicModelProxyHandler = class {
+    constructor() {
+      this.getting = '';
+    }
```

Comment:
Let's use class fields instead (easier to reason about):

```suggestion
    this.getting = '';
```